LEGO1/lego/legoomni/src/infocenter/infocenter.cpp

```diff
@@ -15,20 +43,135 @@ Infocenter::~Infocenter()
 // STUB: LEGO1 0x1006ed90
 MxResult Infocenter::Create(MxDSAction& p_dsAction)
 {
-	return FAILURE;
+	OutputDebugString("create called\n");
```

Comment:
for reference: make sure we remove this before putting into the PR (or do a compiler macro to enable/disable debug strings)